wrapanapi/systems/rhevm.py

```diff
@@ -860,6 +860,27 @@ def list_cluster(self, **kwargs):
         cluster_list = self.api.system_service().clusters_service().list(**kwargs)
         return [cluster.name for cluster in cluster_list]
 
+    def list_disks(self, **kwargs):
+        disks_list = self.api.system_service().disks_service().list()
+        if kwargs:
+            disk_status = kwargs.get('status')
+            if disk_status:
+                disk_status = disk_status.upper()
+
+                if disk_status == 'OK':
+                    return [(disk.name, disk.status) for disk in disks_list
+                    if types.DiskStatus.OK == disk.status]
```

Comment:
You can flatten this by using `getattr(types.DiskStatus, kwargs.get('status').upper()`

Reply:
Note that will raise an AttributeError unless you default it. Combined with the above comment about ValueError raising, you can structure this a few ways. Separate check, or this getattr() call in a try/except that catches AttributeError and raises ValueError. I would prefer the second.